sys/kern/kern_intr.c

```diff
@@ -1519,11 +1519,10 @@ intr_event_handle(struct intr_event *ie, struct trapframe *frame)
 	}
 	critical_exit();
 	td->td_intr_nesting_level--;
-#ifdef notyet
+
 	/* The interrupt is not aknowledged by any filter and has no ithread. */
 	if (!thread && !filter)
 		return (++ie->ie_stray);
-#endif
```

Comment:
I'm quite unsure of this.  This sticks out, but I'm not absolutely sure under what exact condition this is meant to be reenabled.

Reply:
Ah, commit 8381e9f49ec733437754a822ef2e8344115289ac resolves this observation.